components/exercises/chapter-3-oscar/Minimum.tsx

```diff
@@ -0,0 +1,68 @@
+import React, { useState } from 'react';
+import Image from 'next/image';
+import CustomButton from '@/components/CustomButton';
+
+const initialState = [
+  {
+    name: 'Josh',
+    age: 35,
+    image: '/avatar1.png',
+    phrase: 'Hi there! I am Josh, I am 35 years old.',
+  },
+  {
+    name: 'Jacob',
+    age: 23,
+    image: '/avatar2.png',
+    phrase: 'Hi there! I am Jacob, I am 23 years old.',
+  },
+];
+
+const getMinimum = (a: number, b: number) => Math.min(a, b);
```

Comment:
I think the point of the exercise is to create a fn that doesn't use `Math.min`

>The [previous chapter](https://eloquentjavascript.net/02_program_structure.html#return_values) introduced the standard function Math.min that returns its smallest argument. We can build something like that now. Write a function min that takes two arguments and returns their minimum.

Reply:
refactored